CONTRIBUTING.md

```diff
@@ -48,17 +53,69 @@ If the pull request does not have a clear purpose and benefits, it will not be a
-	* HDF5 has a commitment to remaining *machine-independent*; data created on one platform/environment/architecture **must** remain readable by HDF5 on any other.
-	* For binary compatibility, no changes are allowed to public APIs and data structures in the maintenance releases; new APIs can be added.
+    * HDF5 has a commitment to remaining *machine-independent*; data created on one platform/environment/architecture **must** remain readable by HDF5 on any other.
+    * For binary compatibility, no changes are allowed to public APIs and data structures in the maintenance releases; new APIs can be added.
 
 * **New features are documented** - Any new features should have proper documentation; talk to us if you have any questions.
 
+* **When to Write a Release Note** - Generally, a release note must be written for every change that is made to the code for which
+users might see a change in the way the software works. In other words, if a user might see a difference in the way the software works,
+a note should be written. By code we mean the text that will be compiled into one of the company's software products. The code includes
+configuration changes and changes to tools users might work with to configure and build our software.
+
+    * Notes should be added for known problems. Known problems are issues that we know about and have not yet been able to fix.
+
+    * Any change made to address a user-reported problem should be described in a release note.
+
+    * A release note does not need to be written for changes to the code that users will not see. Here are some examples. If you add a
+comment, you do not need to write a release note describing the comment you added. If you rewrite some code to make it read more
+clearly and if there is no change in functionality or performance, then you do not need to write a release note. If you change the
+process by which user software is made, you may not need to write a release note since the change was not made to the code.
+
+    * Users. We have different kinds of users. A release note may be written to be helpful to
+application developers and not system administrators. Users who may find the RELEASE.txt file helpful include the following:
+application developers, library developers, and system administrators.
+
+
+# Release Note <A NAME="releasenote"></A>
+
+* **Entry Syntax**
+The release note entry syntax is shown below.
+
+```
+    - Title/Problem
+
+      Problem/Solution
+
+      Signature
+```
+
+* **Entry Elements** - The elements of the entry - title, problem, solution, and signature - are described in more detail in the table
+below. Descriptions of the problem and the solution should be clear without any ambiguities and should be short without losing clarity or specifics.
+
+    * **Title** - The title or tag should identify one or more categories that will help readers decide if the entry is something they need to study. Can be combined with the `Problem` element
```

Comment:
Should this "**Title**" line be two lines?

Reply:
I don't think there is any indication of where to put a release note - at the top of the appropriate section either under "New Features" or under "Bug Fixes since HDF5-<previous release>", so that entries within a section appear from latest date to earliest date.  This could be in "Where to Write a Release Note" at the bottom?

Reply:
I don't like the title description either!